GOVERNANCE.md

```diff
@@ -53,7 +53,7 @@ and invited to the [private maintainer mailing list](https://lists.cncf.io/g/cnc
 Time zones permitting, Maintainers are expected to participate in the public
 developer meeting, which occurs
 
-Every Monday at 9am EST ([video call link](https://meet.google.com/trx-vqbw-ekb))
+Every two weeks, on Monday, at 9am EST ([video call link](https://meet.google.com/trx-vqbw-ekb)).
```

Comment:
We should also update the meeting link:
```suggestion
Every two weeks, on Monday, at 9am EST ([video call link](https://meet.google.com/kgx-ttwo-zed)).
```

Reply:
Prefer the following change: https://github.com/devfile/api/pull/1381#discussion_r1425444544